### PR TITLE
Fix git ignore and other small updates

### PR DESCRIPTION
**Pull request by dchiu911**
Fix git ignore constant patterns in #1677 and a couple other small updates

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # usethis (development version)
 
+* Add trailing slash to DESCRIPTION URLs created from `use_github_links()` to adhere to new CRAN URL checks
```
Comment (jennybc) [inline NEWS.md:3]: Let's keep the PR more focused on one thing. I'm pretty sure we fixed our slash problems already? E.g. #1526 and #1527. Even if not, it's easier to think about these things separately.

**Review by jennybc — COMMENTED** [review]
> Fix git ignore constant patterns in #1677 and a couple other small updates

The above won't actually close the relevant issue because it's too far from the magic utterance. Also I think this closes a 2nd issue too? (I'm suggesting you split off the 3rd thing)

**Comment by dchiu911** [comment]
I'm using the magic in the commit messages, I think those ones should close the issues? The PR comment is just a general reiteration of what's being done.

File: NEWS.md (hunk 1 above)
Comment (dchiu911) [reply NEWS.md:3]: Sounds good, I'll revert and open another PR. I think the slash problems were only addressed in `code-of-conduct.R`, `logo.R`, `pkgdown.R`

**Pull request closed without merging**